### ISSUE TITLE
Idea

### ISSUE THREAD

**Issue by suomilanittaja** [PhoenicisOrg/phoenicis#1409]
please add proton an alternative to wine.
Sorry bad english!

**Comment by plata** [comment]
I'm not really familliar with Proton, i.e. how it's installed. Might be that it's sufficient to add another Wine source. If that's not the case, we would need a new engine.
Is there anything particular why you would like to have Proton?

Please change the issue title such that it's more clear what the issue is all about.

**Comment by plata** [comment]
From my understanding, most of the Proton improvements are available in Wine as well. So as long as there's no real use case where Proton is better than plain Wine, I don't really see a point supporting it.

**Comment by ImperatorS79** [comment]
Proton inculdes DXVK by default (better d3dx11 performance) and also the wine-esync patches, which also improves performance but are not in regular wine.

Also you should note that steam will mark as "linux supported" all games that work under Proton.

Any news about the missing dependencies in phoenicis wine build ?

**Comment by plata** [comment]
Ok. As I've said above it might be enough to add an additional Wine source. Are Proton binaries available somewhere?
The Wine builds didn't really change yet. Maybe try the Lutris Wine or official Wine sources.

**Comment by ImperatorS79** [comment]
You could try to replace current phoenics wine builds by wine builds from [https://dl.winehq.org/wine-builds/ubuntu/](https://dl.winehq.org/wine-builds/ubuntu/).

Here is the dependency graph of the winehq-devel package (wine 64/32) for bionic (winehq-devel.deb = winehq-devel_3.17.0~bionic_amd64.deb) : 

winehq-devel_3.17.0~bionic_amd64.deb
------------------------------------
 _Pre-Depends_: dpkg (>= 1.14.12ubuntu3)
 _Depends_: wine-devel (= 3.17.0~bionic)
 _Contains_: all bin, share: applications, doc, man

wine-devel_3.17.0~bionic_amd64.deb
----------------------------------
 _Pre-Depends_: dpkg (>= 1.14.12ubuntu3)
 _Depends_: libc6 (>= 2.17), wine-devel-amd64 (= 3.17.0\~bionic), wine-devel-i386 (= 3.17.0~bionic)
 _Contains_: all bin but not wine64, wine64-preloader, wine, wine-preloader; share: appications, man, wine
 
wine-devel-amd64_3.17.0~bionic_amd64.deb
---------------------------------------- 
 _Pre-Depends_: dpkg (>= 1.14.12ubuntu3)
 _Depends_: libasound2 (>= 1.0.16), libc6 (>= 2.27), libgcc1 (>= 1:3.0), libglib2.0-0 (>= 2.12.0), libgphoto2-6 (>= 2.5.10), libgphoto2-port12 (>= 2.5.10), libgstreamer-plugins-base1.0-0 (>= 1.0.0), libgstreamer1.0-0 (>= 1.4.0), liblcms2-2 (>= 2.2+git20110628), libldap-2.4-2 (>= 2.4.7), libmpg123-0 (>= 1.6.2), libopenal1 (>= 1.14), libpcap0.8 (>= 0.9.8), libpulse0 (>= 0.99.1), libudev1 (>= 183), libx11-6, libxext6, libxml2 (>= 2.9.0), zlib1g (>= 1:1.1.4), libasound2-plugins, libncurses5
 _Recommends_: libcapi20-3, libcups2 (>= 1.4.0), libdbus-1-3 (>= 1.9.14), libfontconfig1 (>= 2.12), libfreetype6 (>= 2.2.1), libglu1-mesa | libglu1, libgnutls30 (>= 3.5.0), libgsm1 (>= 1.0.13), libgssapi-krb5-2 (>= 1.6.dfsg.2), libjpeg8 (>= 8c), libkrb5-3 (>= 1.6.dfsg.2), libncurses5 (>= 6), libodbc1 (>= 2.3.1), libosmesa6 (>= 10.2~), libpng16-16 (>= 1.6.2-1), libsane1 (>= 1.0.24), libsdl2-2.0-0 (>= 2.0.8), libtiff5 (>= 4.0.3), libv4l-0 (>= 0.5.0), libvulkan1, libxcomposite1 (>= 1:0.3-1), libxcursor1 (>> 1.1.2), libxfixes3, libxi6, libxinerama1, libxrandr2, libxrender1, libxslt1.1 (>= 1.1.25), libxxf86vm1
 _Contains_: wine64 and wine64-preloader, lib64
 
wine-devel-i386_3.17.0~bionic_i386.deb
--------------------------------------
 _Pre-Depends_: dpkg (>= 1.14.12ubuntu3)
 _Depends_: libasound2 (>= 1.0.16), libc6 (>= 2.27), libglib2.0-0 (>= 2.12.0), libgphoto2-6 (>= 2.5.10), libgphoto2-port12 (>= 2.5.10), libgstreamer-plugins-base1.0-0 (>= 1.0.0), libgstreamer1.0-0 (>= 1.4.0), liblcms2-2 (>= 2.2+git20110628), libldap-2.4-2 (>= 2.4.7), libmpg123-0 (>= 1.13.7), libopenal1 (>= 1.14), libpcap0.8 (>= 0.9.8), libpulse0 (>= 0.99.1), libudev1 (>= 183), libx11-6, libxext6, libxml2 (>= 2.9.0), zlib1g (>= 1:1.1.4), libasound2-plugins, libncurses5
 _Recommends_: libcapi20-3, libcups2 (>= 1.4.0), libdbus-1-3 (>= 1.9.14), libfontconfig1 (>= 2.12), libfreetype6 (>= 2.2.1), libglu1-mesa | libglu1, libgnutls30 (>= 3.5.0), libgsm1 (>= 1.0.13), libgssapi-krb5-2 (>= 1.6.dfsg.2), libjpeg8 (>= 8c), libkrb5-3 (>= 1.6.dfsg.2), libncurses5 (>= 6), libodbc1 (>= 2.3.1), libosmesa6 (>= 10.2~), libpng16-16 (>= 1.6.2-1), libsane1 (>= 1.0.24), libsdl2-2.0-0 (>= 2.0.8), libtiff5 (>= 4.0.3), libv4l-0 (>= 0.5.0), libvulkan1, libxcomposite1 (>= 1:0.3-1), libxcursor1 (>> 1.1.2), libxfixes3, libxi6, libxinerama1, libxrandr2, libxrender1, libxslt1.1 (>= 1.1.25), libxxf86vm1
 _Contains_: wine, wine-preloader, lib, share: man

So you could build your {bin, lib64, lib, share} directory by taking bin from winehq-devel_3.17.0\~bionic_amd64.deb, share from wine-devel_3.17.0\~bionic_amd64.deb, lib64 from wine-devel-amd64_3.17.0\~bionic_amd64.deb and lib from wine-devel-i386_3.17.0\~bionic_i386.deb. By taking wine for linux and macOS directly from the wine website, you don't have to bother with that anymore.

And phoenicis should be installed with all the libs which wine depends (in Depends AND Recommends field) (sorry for the long post).

**Comment by plata** [comment]
Could work but honestly, I would try https://github.com/PhoenicisOrg/wine-sources/blob/master/lutris-linux.json first.

**Comment by ImperatorS79** [comment]
Yes, that just that if lutris stops uploading wine versions, then users do not have wine versions anymore.

**Comment by madoar** [comment]
@qparis  this sounds like a feature for the new [buildsystem](https://github.com/PhoenicisOrg/phoenicis-winebuild), right?

**Comment by qparis** [comment]
Yes, the ticket already exists. The problem so far is that proton seems to require to be built inside docker, so docker inside docker is not optimal. But I think we can do it

**Comment by madoar** [comment]
Can we close this issue then?
